### ISSUE TITLE
App Invites

### ISSUE THREAD

**Issue by devsead** [Wizcorp/phonegap-facebook-plugin#1040]
Hi,

Is it possible to use app invites (https://developers.facebook.com/docs/app-invites) with phonegap-facebook-plugin ?

Thanks

**Comment by roysG** [comment]
I also would like to know that, is it possible ?

**Comment by toptierlabs** [comment]
I am also trying to do this.

Any?

**Comment by omaxlive** [comment]
I need it too, is it possible?

**Comment by chris08002** [comment]
You can use app request. I think it is not the same, but works.

```
var options = {
    method: "apprequests",
   message: "Some text" 
}
$cordovaFacebook.showDialog(options).then(fbPostSuccess, fbPostError);
```

The receiving user gets directly from the notification to the app store.

Any better solution than this?

**Comment by MartinKip** [comment]
That option only works if your facebook application is a game application. Normal applications wont work

**Comment by promactkaran** [comment]
Yes, only if the app is categorised as GAME, then only it works. Is there any solution for Normal apps using this plugin ?

**Comment by itconsulting-live** [comment]
Aly luck with this, anybody?

**Comment by quedicesebas** [comment]
Needed...I think first we need to upgrade the plugin to the v4 SDK

**Comment by vexsoluciones** [comment]
hello ??? i need the same please advise...

**Comment by BojDom** [comment]
+1 i think everyone want to make user to share their apps, why anyone haven't a solution yet?

**Comment by chris08002** [comment]
You can use the facebook4 SDK fork: https://github.com/jeduan/cordova-plugin-facebook4.git
Works fine with app invites

**Comment by vexsoluciones** [comment]
it's working fine with ios??

**Comment by chris08002** [comment]
Actually not :( Just on Android. I mean the plugin in general works, but not the app invite, at least not in our implementation. I only had tested android before....

**Comment by Tanmay2905** [comment]
@chris08002 I have integrated this plugin.Followed all steps. The plugin sends app-invite to friends, but people to whom I sent the app-invite did not get a invite or notification.
Thank You.

**Comment by Tanmay2905** [comment]
@chris08002 my facebook app is in public state
